### DISCUSSION TITLE
How to deal with spiderwebs?

### DISCUSSION THREAD

**Discussion by atv2016** [blakeblackshear/frigate#8130] [General]
This is becoming a problem lol and causing false detections.

Does anyone have any long lasting solutions that would deter spiders? I read about spiderex but not sure if that works?

would love to hear if anyone else had luck with this.

**Comment by NickM-27** [comment]
So far I have had good results submitting them as false positives and on my latest model I have noticed spider webs in the camera view but no false positives https://github.com/blakeblackshear/frigate/discussions/7932

**Comment by atv2016** [comment]
Cool that’s good to know!
________________________________

**Reply by NickM-27** [reply]
otherwise, bugs are generally attracted to IR cameras so color starlight cameras can help in that regard as well

**Comment by gilesm2** [comment]
It would be handy if Frigate raised a warning if "significant" cobwebs were found - sometimes you don't notice them for days/weeks/months if the system is running smoothly.

**Reply by adamburgoyne** [reply]
This is probably too specific a use case to build into Frigate but I think I've solved this with the LLM HomeAssistant integration. I have an automation that runs once a day and answers the question _"Are there any cobwebs on the camera? Answer with yes or no.",_ sending a notification to clean if yes.

Works well so far and I get a notification every couple of months or so.
